content/sensu-go/5.10/reference/datastore.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Datastore"
+description: "Sensu stores the most recent event for each entity and check pair using an embedded etcd (default) or an external etcd instance. Sensu also supports using an external PostgreSQL instance for event storage in place of etcd (licensed-tier only). Read the reference to configure enterprise-scale event storage using PostgreSQL."
+weight: 10
+version: "5.10"
+product: "Sensu Go"
+menu:
+  sensu-go-5.10:
+    parent: reference
+---
+
+- [Event storage](#event-storage)
+- [Scaling event storage](#scaling-event-storage) (licensed-tier only)
+  - [Requirements](#requirements)
+  - [Configuration](#configuration)
+  - [Specification](#specification)
+
+## Event storage
+
+Sensu stores the most recent event for each entity and check pair using an embedded etcd (default) or an [external etcd][8] instance.
+You can access event data using the [dashboard][9] events page, [`sensuctl event` commands][10], and the [events API][11].
+To store event data over time, we recommend integrating Sensu with a time series database, like [InfluxDB][12].
+
+## Scaling event storage
+
+**LICENSED TIER**: Unlock enterprise-scale event storage in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][13].
+
+Sensu supports using an external PostgreSQL instance for event storage in place of etcd.
+PostgreSQL can handle significantly higher volumes of Sensu events, letting you scale Sensu beyond etcd's 8GB limit.
+
+Configured with a PostgreSQL event store, Sensu connects to PostgreSQL to store and retrieve event data in place of etcd, while etcd continues to store Sensu configuration data.
+You can access event data stored in PostgreSQL using the same Sensu web UI, API, and sensuctl processes as etcd-stored events.
+
+### Requirements
+
+Sensu supports PostgreSQL 9.5 and later as well as [Amazon Relational Database Service][3] (Amazon RDS).
+See the [PostgreSQL docs][14] to install and configure PostgreSQL.
+
+### Configuration
+
+At the time of enabling the PostgreSQL event store, event data cuts over from etcd to PostgreSQL, resulting in a loss of recent event history.
+No restarts or Sensu backend configuration changes are required to enable the PostgreSQL event store.
+
+_WARNING: At this time, the PostgreSQL event store cannot be disabled once configured._
```

Comment:
Should we mention that we plan to address this in the subsequent release?

Reply:
Usually we try to avoid that sort of info in the docs, but I think in this case, a friendly callout would be helpful. Good callout!